tests/unit/amo/components/TestHomepageShelves.js

```diff
@@ -146,6 +148,22 @@ describe(__filename, () => {
     },
   );
 
+  it('passes addonInstallSource as tag-shelf-{tag} when endpoint is random-tag', () => {
+    const url =
```

Comment:
We try to use constants in tests to avoid repeating the same hardcoded value in the test. So `const tag = 'foo'`, and then use `tag` both when creating the `url` and in the assertion.